pkg/applicationserver/io/nats/nats.go

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package nats implements the NATS frontend.
+package nats
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	nats "github.com/nats-io/nats.go"
+	"go.thethings.network/lorawan-stack/pkg/applicationserver/io"
+	iopubsub "go.thethings.network/lorawan-stack/pkg/applicationserver/io/pubsub"
```

Comment:
The rename is unnecessary here - I've forgot to remove it while refactoring away from the `go-cloud/pubsub` package.